src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2295,6 +2295,7 @@ because xbuild doesn't support framework reference assemblies.
 	Debug="$(AndroidIncludeDebugSymbols)"
 	MultiDex="$(AndroidEnableMultiDex)"
 	NeedsInternet="$(AndroidNeedsInternetPermission)"
+	InstantRunEnabled="$(_InstantRunEnabled)"
```

Comment:
@jonathanpeppers does this property have a default of `False`  when using only XA? I ask because if its empty MSBuild will throw and error when converting to the bool the Task needs.

Reply:
Ah it does. cool. so ignore me